### ISSUE TITLE
Status text on "Publish" button not announced by screenreader

### ISSUE THREAD

**Issue by outofambit** [desktop/desktop#6581]
While screenreader announces "Pushing to origin Hang on, unavailable" after activation the dynamic status ("Publishing to Origin, hang on", "Publishing to origin, Total 0 (delta 0), reused 0 (delta...", "Fetching origin, hand on" and "Refreshing Repository Hang on...") are not announced. Final state can be found if the focus is returned - "Fetch origin last fetched just now" - but could be confusing as they process started with Publish and has ended with Fetch. This button also has a built in progress bar via a shading effect. % complete should be announced to Screenreader users as well.

**Comment by tidy-dev** [comment]
This was resolved in https://github.com/desktop/desktop/pull/16985
